### PR TITLE
feat: add new municipalities and OCMWs for 2025 mergers

### PR DESCRIPTION
**Pull request by mirdono**
## Overview
As part of the municipality mergers for 2025, four new municipalities and their
corresponding OCMWs will have new KBO numbers. Consequently, these organizations
need to be fully manually created in OP.

This commit adds the migrations to import the organisations for the merged
municipalities and OCMWs such that their URIs can already be shared among
applications.

### Core data
Provided by business:
- name
- legal name
- region (only for municipalities)
- status, set to "In oprichting" (Eng. "In formation")
- KBO number
- OVO number
- Province

Furthermore, the necessary triples are created allow adding a SharePoint
identifier later on.

Note: The NIS-codes (and associated data) are not imported by these
migrations. This data will be added via future migrations when the NIS2025
update has been released.

### Governing bodies
For each new organisation the necessary governing bodies are added, each with
2025-01-01 as starting date.

Note, for municipalities more kinds of governing bodies were added in order to
be consistent with already existing data.

- Municipalities:
  - Burgemeester
  - Gemeenteraad
  - College van Burgemeester en Schepenen
  - Adjunct-financieel directeur (not specified in ticket)
  - Adjunct-algemeen directeur (not specified in ticket)
  - Financieel directeur (not specified in ticket)
  - Algemeen directeur (not specified in ticket)

- OCMWs:
  - Voorzitter van het Bijzonder Comité voor de Sociale Dienst
  - Raad voor Maatschappelijk Welzijn
  - Bijzonder Comité voor de Sociale Dienst
  - Vast Bureau

## Notes:
- Finding the new organizations via the filters on the index page requires
  re-indexing
- Currently missing data, such as addresses and contact details, will be added
  by business in the future.
- The script used to generate the migrations importing the new organisations was
  attached in the ticket.
- The migration relating the new municipalities and OCMWs was manually written.

## Related tickets
OP-3269

### CODE REVIEW

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql:15]: This insert has to be `INSERT DATA`. I think the spec does not allow a lone INSERT.

The other `xxx-merged-yyy.sparql` also have the same issue in the same location; I'll add a quick comment on each one.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122840-merged-municipality-nazareth_de_pinte.sparql [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122840-merged-municipality-nazareth_de_pinte.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122841-merged-municipality-pajottegem.sparql [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122841-merged-municipality-pajottegem.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122842-merged-municipality-merelbeke_melle.sparql [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122842-merged-municipality-merelbeke_melle.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122843-merged-ocmw-beveren_kruibeke_zwijndrecht.sparql [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122843-merged-ocmw-beveren_kruibeke_zwijndrecht.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122844-merged-ocmw-nazareth_de_pinte.sparql [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122844-merged-ocmw-nazareth_de_pinte.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122845-merged-ocmw-pajottegem.sparql [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122845-merged-ocmw-pajottegem.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122846-merged-ocmw-merelbeke_melle.sparql [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```
Comment (Riadabd) [inline config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122846-merged-ocmw-merelbeke_melle.sparql:15]: This insert has to be `INSERT DATA`.

File: config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql (hunk 1 above)
Comment (mirdono) [reply config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql]: Interestingly a line `INSERT` does work, but you are right that an explicit `INSERT DATA` is better here.

Note: I regenerated the migrations so the filenames (and UUIDs) changed in the process.

Comment (Riadabd) [reply config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql]: Yes indeed, it seems `virtuoso` accepts the lone insert (probably transforms it to an insert data internally).

**Review by Riadabd — APPROVED** [review]
The merged organizations show up as expected :+1:

**Pull request merged**